packages/server/seeds/dev/ref/users.js

```diff
@@ -25,7 +25,7 @@ module.exports = [
     },
     {
         id: 2,
-        email: 'mindy@usdigitalresponse.org', // fake email for testing
+        email: 'mhuang@usdigitalresponse.org', // fake email for testing
```

Comment:
Note this will break some tests that refer to this email; be sure to grep for those and update the email there. See #63 and #67. #67 is to make this type of breakage less likely.